GPU/Common/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if(ALIGPU_BUILD_TYPE STREQUAL "O2")
                              GPUCA_TPC_GEOMETRY_O2 HAVE_O2HEADERS)
   
   # cuda test, only compile if CUDA
-  if(CUDA_ENABLED AND NOT $ENV{ALIBUILT_O2_TESTS})
+  if(ENABLE_CUDA AND NOT ENV{ALIBUILD_O2_TESTS})
```

Comment:
after  discussion with @ktf we decided that this is not the right approach. Actually, we want to build the tests if we have CUDA and HIP available but exclude them from running when no cards are present, which can be done in the O2 recipe.